pkg/subscriptions/subscriptions.js

```diff
@@ -387,6 +390,10 @@ require([
             args.push('--username=' + username);
             args.push('--password=' + password);
 
+            // only pass org info if user provided it
+            if (org)
+                args.push('--org=' + org);
```

Comment:
Does subscription-manager support having this as two arguments? ie: 

```
if (org) {
    args.push('--org');
    args.push(org);
}
```

Because if so, it would help ensure that the arguments passed without escaping went through correctly.